### DISCUSSION TITLE
不知道为啥连接不了模拟器

### DISCUSSION THREAD

**Discussion by okitaccc** [MaaAssistantArknights/MaaAssistantArknights#7161] [Q&A] [answered]
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/57834064/6b81ed12-e9c0-4ac1-a6db-0ef6493c6c85)
MAA报错如上所示，前几天模拟器还能正常运行的，4.26.0更新之后就不行了呃呃

**Comment by ABA2396** [comment]
去 #7162 发一下 debug/asst.log 和 debug/gui.log

**Reply by okitaccc** [reply]
[asst.log](https://github.com/MaaAssistantArknights/MaaAssistantArknights/files/13238826/asst.log)
[gui.log](https://github.com/MaaAssistantArknights/MaaAssistantArknights/files/13238827/gui.log)
（注：之前看4.26.0用不了就删了重下了一次，这个是重下之后的）

**Reply by ABA2396** [reply, marked as answer]
说了到 #7162 发

一般error: device offline都是模拟器出问题了按照红字操作过（重启模拟器和adb）或者（重启电脑）吗
